### ISSUE TITLE
Code display plugin on ClassicPress.net not working

### ISSUE THREAD

**Issue by klein-the-donkey** [ClassicPress/ClassicPress-Network#64]
Is the plugin broken? I can't see the button in the editor and the code that's normally displayed like:
![Schermafbeelding 2019-11-06 om 16 56 04](https://user-images.githubusercontent.com/52501540/68315586-48961800-00b8-11ea-920b-aa9c1488b2a6.png)
Is now displayed like:
![Schermafbeelding 2019-11-06 om 16 55 42](https://user-images.githubusercontent.com/52501540/68315597-4e8bf900-00b8-11ea-9745-ab45bccab697.png)

**Comment by timbocode** [comment]
As I recall, James had to remove the `Crayon Syntax Highlighter` plugin because it's not PHP7 compatible and was throwing numerous errors. @nylen should be able to confirm.

**Comment by nylen** [comment]
Yes, that's right. It's also pretty over-large for what it was doing. I'd be open to installing a lightweight replacement if you know of one.

**Comment by johnalarcon** [comment]
I've worked on the plugin to bring it to PHP 7 compatibility. Also, if you remove the 50 needless languages, it's much slimmer; this involves just removing the directories. For the ClassicPress blog, leaving in only html, xml, apache, php, sql, css, javascript, and json would probably cover every last need.

**Comment by johnalarcon** [comment]
I've built a slim and performant (250K footprint, 55K output) [Syntax Highlighter](https://github.com/codepotent/syntax-highlighter/issues/1) plugin based on [Prism](https://prismjs.com). It still has one issue, but, may become a viable option for ClassicPress.net. Please give it a test-drive.

@nylen, I proposed a solution on [this issue](https://github.com/codepotent/syntax-highlighter/issues/1); is this the correct approach?
